sdk/webpubsub/azure-messaging-webpubsub/README.md

```diff
@@ -0,0 +1,179 @@
+```java
+webPubSubServiceClient.sendToAll("Hello world!");
+```
+
+### Broadcast message to a group
+
+<!-- embedme ./src/samples/java/com/azure/messaging/webpubsub/ReadmeSamples.java#L67-L68 -->
+```java
+WebPubSubGroup javaGroup = webPubSubServiceClient.getGroup("Java");
+javaGroup.sendToAll("Hello Java!");
+```
+
+### Send message to a connection
+
+<!-- embedme ./src/samples/java/com/azure/messaging/webpubsub/ReadmeSamples.java#L80-L80 -->
+```java
+webPubSubServiceClient.sendToConnection("myconnectionid", "Hello connection!");
+```
+
+### Send message to a user
+<!-- embedme ./src/samples/java/com/azure/messaging/webpubsub/ReadmeSamples.java#L92-L92 -->
+```java
+webPubSubServiceClient.sendToUser("Andy", "Hello Andy!");
+```
+
+## Troubleshooting
+
+### Enable client logging
+You can set the `AZURE_LOG_LEVEL` environment variable to view logging statements made in the client library. For
+example, setting `AZURE_LOG_LEVEL=2` would show all informational, warning, and error log messages. The log levels can
+be found here: [log levels][log_levels].
+
+### Default HTTP Client
+All client libraries by default use the Netty HTTP client. Adding the above dependency will automatically configure
+the client library to use the Netty HTTP client. Configuring or changing the HTTP client is detailed in the
+[HTTP clients wiki](https://github.com/Azure/azure-sdk-for-java/wiki/HTTP-clients).
+
+### Default SSL library
+All client libraries, by default, use the Tomcat-native Boring SSL library to enable native-level performance for SSL
+operations. The Boring SSL library is an uber jar containing native libraries for Linux / macOS / Windows, and provides
+better performance compared to the default SSL implementation within the JDK. For more information, including how to
+reduce the dependency size, refer to the [performance tuning][performance_tuning] section of the wiki.
+
+## Next steps
+
+- Samples are explained in detail [here][samples_readme].
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to
+a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights
+to use your contribution.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate
+the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to
+do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see
+the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or
+comments.
+
+<!-- LINKS -->
+
+[source_code]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/webpubsub/azure-messaging-webpubsub/src
+[product_documentation]: https://docs.microsoft.com/azure/azure-signalr/signalr-overview
```

Comment:
do we have another product page for WebPubSub? or just share the same one with SignalR?

Reply:
I am not aware of another public product page for Web PubSub. @vicancy can you please let me know if there's an alternate link that can be used here?

Reply:
We can use this short link https://aka.ms/awps/doc